### ISSUE TITLE
Tasks not generated for jurors added to ranking round

### ISSUE THREAD

**Issue by papuass** [hatnote/montage#109]
Tasks are not generated for jurors I added to a ranking round later. Two original jurors have their 30 images to order, but jurors added later have zero tasks.

Also "Quorum" is confusing here as it can only be the same number as number of jurors.

My campaign is "Wiki Loves Monuments 2017 LV", round "Uzvarētāju noteikšana".

I (Papuass) will not vote (added myself just to check if it works). I want to add one more juror.

**Comment by leilaz** [comment]
@papuass I'm looking at it. Was your quorum originally 2 and you changed it to 4?

**Comment by papuass** [comment]
Yes, I changed the quorum from 2 to 3, then to 4. At that point not sure why.

**Comment by leilaz** [comment]
@papuass ok, this is the problem. The tool should technically doesn't let you increase quorum cuz we don't know how to handle it, yet. Sorry about this. Here is what I recommend: Download the result of votes for the one juror that has completed their voting. Delete this round and create a new round with the 3 jurors you want in and set the quorum to 3. You then have to download the result of this round and merge it with the result of the votes from the 1 juror you are handling separately.

**Comment by papuass** [comment]
OK, I deleted the old round and created a new one.

Noticed that description was not saved, as per issue #102
